packages/playground/src/utils/delete_deployment.ts

```diff
@@ -3,7 +3,7 @@ import type { GridClient } from "@threefold/grid_client";
 import { ProjectName } from "@/types";
 
 import { loadVM } from "./deploy_vm";
-import { getSubdomain } from "./gateway";
+import { getSubdomain, loadDeploymentGateways } from "./gateway";
```

Comment:
Shouldn't this function change to loadDeploymentDomains?

Reply:
No, because we're using name domain for only fvm/vm but reset solution using gateway